src/content-layout/styles.scss

```diff
@@ -53,14 +53,9 @@ nodes will directly touch with no gap between them.
   }
 
   &:not(.has-header) {
-    grid-template-rows:
-      auto calc(
-        #{awsui.$space-dark-header-overlap-distance} + var(#{custom-props.$containerFirstOverlapExtension}, 0px) + var(#{custom-props.$containerFirstGap}, 0px)
-      )
-      1fr;
-
+    grid-template-rows: auto calc(awsui.$space-dark-header-overlap-distance + awsui.$space-xs + awsui.$space-scaled-xxs) 1fr;
     > .content {
-      padding-block-start: var(#{custom-props.$containerFirstGap}, 0px);
+      padding-block-start: calc(awsui.$space-xs + awsui.$space-scaled-xxs);
```

Comment:
Basically, this is the change. Before it was 4px or 12px based on app layout configuration. Now it is always 12px regardless what is happening in app layout.